src/blockfield.js

```diff
@@ -9,21 +9,21 @@ module.exports = function blockfield() {
     redact = Parser.prototype.options.redact;
     Parser.prototype.inlineTokenizers[BLOCKFIELD] = tokenizeResourcelink;
 
-    if (Parser.prototype.restorationMethods) {
-      Parser.prototype.restorationMethods[BLOCKFIELD] = function (add, node) {
-        return add({
-          type: 'rawtext',
-          value: `{${node.redactionData}}`
-        });
-      };
-    }
-
     // Run it just before `html`.
     const methods = Parser.prototype.inlineMethods;
     methods.splice(methods.indexOf('html'), 0, BLOCKFIELD);
   }
 };
 
+module.exports.restorationMethods = {
+  blockfield: function(node) {
```

Comment:
nit: (here and elsewhere)
```suggestion
  [BLOCKFIELD]: function(node) {
```